### PR TITLE
Cancellation reason in Github comments

### PR DESCRIPTION
**Pull request by g4s8**
#499 - added `cancel_order` claim reason to Github comment message, the text for a reason is already defined, e.g. in `resign_on_delay`: "It is older than %d day(s), see §8".

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#642](https://github.com/zerocracy/farm/issues/642) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#642](https://github.com/zerocracy/farm/issues/642) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/560616389). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/main/resources/com/zerocracy/stk/pm/in/orders/cancel_order.groovy [hunk 1]
```diff
@@ -43,10 +43,16 @@ def exec(Project project, XML xml) {
...
+  if (claim.hasParam('reason')) {
+    reason = claim.param('reason')
+  } else {
+    reason = 'Order was cancelled'
+  }
   claim.reply(
     new Par(
-      'The user @%s resigned from %s, please stop working',
-    ).say(performer, job)
+      'The user @%s resigned from %s, please stop working: %s',
```
Comment (carlosmiranda) [inline src/main/resources/com/zerocracy/stk/pm/in/orders/cancel_order.groovy]: @g4s8 I'd rephrase it as such: `The user @%s resigned from %s, please stop working. Reason for job resignation: %s`

**Comment by g4s8** [comment]
@carlosmiranda I changed it, thanks. Please see new changes.

**Comment by carlosmiranda** [comment]
@yegor256 looks good to merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), total is [+1590](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
The job [#642](https://github.com/zerocracy/farm/issues/642) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+10884](https://www.0crat.com/u/yegor256)
